### PR TITLE
Add security page to properties

### LINKED ISSUE #1647: Add security tab to properties
From a feedback on Feedback Hub (Apps->Files UWP).
https://aka.ms/AA96qf4

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- For #1647

**Details of Changes**
Add details of changes here.
- Adds security tab to properties window

**TODO**
- [x] Persist changed permissions to filesystem
- [ ] ~Implement "advanced permission options"~ -> next PR
- [x] UI polish

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by gave92** [comment]
Basic page is working!

image

**Comment by yaichenbaum** [comment]
> Basic page is working!
> 
> image

@gave92 That looks great!

File: Files/Views/Pages/PropertiesSecurity.xaml [hunk 1]
```diff
@@ -95,15 +95,25 @@
                     Style="{StaticResource PropertyName}"
                     Text="Group or user names" />
 
-                <CommandBar
+                <StackPanel
```
Comment (gave92) [inline Files/Views/Pages/PropertiesSecurity.xaml:99]: @yair100 note that there is another command bar you may wish to change, line 75

**Review by yaichenbaum — APPROVED** [review]
Great work! LGTM

File: Files/Views/Pages/PropertiesSecurity.xaml (hunk 1 above)
Comment (yaichenbaum) [reply Files/Views/Pages/PropertiesSecurity.xaml:99]: @gave92 Taken care of, thanks!

**Pull request merged**